src/test/unit_tests/test_aws_sts.js

```diff
@@ -0,0 +1,65 @@
+/* Copyright (C) 2020 NooBaa */
+'use strict';
+
+const mocha = require('mocha');
+const assert = require('assert');
+const sinon = require('sinon');
+const AWS = require('aws-sdk');
+const cloud_utils = require('../../util/cloud_utils');
+const dbg = require('../../util/debug_module')(__filename);
+const fs = require("fs");
+
+mocha.describe('AWS STS tests', function() {
```

Comment:
If you want this to run as part of the unit tests you should add it to `src/test/unit_tests/index.js`